x-pack/plugins/apm/public/hooks/useAvgDurationByBrowser.ts

```diff
@@ -43,14 +43,13 @@ export function useAvgDurationByBrowser() {
             query: {
               start,
               end,
-              transactionName,
```

Comment:
Transaction name is not used in the query.

Reply:
Afaict it's used when you navigate to a transaction group.

Reply:
This is the request my browser made:
```
/api/apm/services/opbeans-java/transaction_groups/breakdown?start=2020-07-21T13%3A52%3A34.402Z&end=2020-07-21T14%3A07%3A34.402Z&transactionName=DispatcherServlet%23doGet&transactionType=request&uiFilters=%7B%22serviceName%22%3A%5B%22opbeans-java%22%5D%7D
```

Reply:
Oh, sorry, I was looking at breakdown.

Reply:
I think you've hit a bug. This most definitely should take transaction name and type into account.

Reply:
transaction type is hardcoded so we don't have to worry about that:
https://github.com/elastic/kibana/blob/57d67792b4f1388b71dbc3e27cc6e168eacab5dc/x-pack/plugins/apm/server/lib/transactions/avg_duration_by_browser/fetcher.ts#L32

But we should add a terms filter for transaction name